interview-questions.md

```diff
@@ -15,45 +15,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 2. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: This is the type of database that stores and provides access to data that is related. There are non-relational databases as well.
 
-  Researched answer:
+  Researched answer: Non-relational databases, also called NoSQL databases, the most popular being MongoDB, DocumentDB, Cassandra, Coachbase, HBase, Redis, and Neo4j. 
 
 
 
 3. What is object-relational mapping?
 
-  Your answer:
+  Your answer: Converting data from a database using object oriented programming. 
 
-  Researched answer:
+  Researched answer: Object-Relational Mapping (ORM) is a technique that lets you query and manipulate data from a database using an object-oriented paradigm.
 
 
 
 4. What is a gem?
 
-  Your answer:
+  Your answer: Library with functions, extention.
```

Comment:
Okay this is the 5th Week of interview questions. I know these seem like they're able to be breezed over but the intention behind these questions is to prepare you for interviews. Let's focus on leaving substantial information for your future self that you can review when you are heading into interviews.